core/navigation/src/main/java/com/iteneum/navigation/graphs/DashBoardNavGraph.kt

```diff
@@ -14,7 +14,7 @@ fun NavGraphBuilder.dashboardNavGraph(navController: NavHostController) {
         startDestination = DashBoardScreen.Profile.route
     ) {
         composable(route = DashBoardScreen.Profile.route) {
-            ProfileView()
+            ProfileView(navigation = { /*TODO*/ })
```

Comment:
Explain in that TODO what you need to do in that specific part.

Reply:
Got it! Done!